dev/io.openliberty.jakarta.cdi.4.0/bnd.bnd

```diff
@@ -20,41 +20,38 @@ Bundle-SymbolicName: io.openliberty.jakarta.cdi.4.0; singleton:=true
+ jakarta.enterprise.context.control;version="4.0";uses:="jakarta.enterprise.context,jakarta.interceptor",\
+ jakarta.enterprise.context.spi;version="4.0",\
+ jakarta.enterprise.event;version="4.0";uses:="jakarta.enterprise.util",\
+ jakarta.enterprise.inject;version="4.0";uses:="jakarta.enterprise.context,jakarta.enterprise.inject.spi,jakarta.enterprise.util,jakarta.inject",\
+ jakarta.enterprise.inject.build.compatible.spi;version="4.0";uses:="jakarta.enterprise.context.spi,jakarta.enterprise.event,jakarta.enterprise.inject,jakarta.enterprise.inject.spi,jakarta.enterprise.lang.model,jakarta.enterprise.lang.model.declarations,jakarta.enterprise.lang.model.types",\
+ jakarta.enterprise.inject.literal;version="4.0";uses:="jakarta.enterprise.util,jakarta.inject",\
+ jakarta.enterprise.inject.se;version="4.0";uses:="jakarta.enterprise.inject,jakarta.enterprise.inject.spi",\
+ jakarta.enterprise.inject.spi;version="4.0";uses:="jakarta.el,jakarta.enterprise.context.spi,jakarta.enterprise.event,jakarta.enterprise.inject,jakarta.enterprise.inject.spi.configurator,jakarta.interceptor",\
+ jakarta.enterprise.inject.spi.configurator;version="4.0";uses:="jakarta.enterprise.context.spi,jakarta.enterprise.event,jakarta.enterprise.inject,jakarta.enterprise.inject.spi,jakarta.enterprise.util",\
+ jakarta.enterprise.util;version="4.0",\
+ jakarta.enterprise.lang.model;version="4.0";uses:="jakarta.enterprise.lang.model.declarations,jakarta.enterprise.lang.model.types", \
+ jakarta.enterprise.lang.model.declarations;version="4.0";uses:="jakarta.enterprise.lang.model,jakarta.enterprise.lang.model.types", \
+ jakarta.enterprise.lang.model.types;version="4.0";uses:="jakarta.enterprise.lang.model,jakarta.enterprise.lang.model.declarations", \
+ jakarta.inject;version="2.0.1"
 
-#these imports are copied from the manifest files of the original bundles. The only addition is for org.osgi.framework.
+#these imports are copied from the manifest files of the original bundles.
 Import-Package: \
  jakarta.el;version="4.0",\
- jakarta.enterprise.context;version="[3.0,4)",\
- jakarta.enterprise.context.spi;version="[3.0,4)",\
- jakarta.enterprise.event;version="[3.0,4)",\
- jakarta.enterprise.inject;version="[3.0,4)",\
- jakarta.enterprise.inject.spi;version="[3.0,4)",\
- jakarta.enterprise.inject.spi.configurator;version="[3.0,4)",\
+ jakarta.enterprise.context;version="[4.0,5)",\
+ jakarta.enterprise.context.spi;version="[4.0,5)",\
+ jakarta.enterprise.event;version="[4.0,5)",\
+ jakarta.enterprise.inject;version="[4.0,5)",\
+ jakarta.enterprise.inject.spi;version="[4.0,5)",\
+ jakarta.enterprise.inject.spi.configurator;version="[4.0,5)",\
  jakarta.enterprise.lang.model;version="[4.0,5)",\
  jakarta.enterprise.lang.model.declarations;version="[4.0,5)",\
  jakarta.enterprise.lang.model.types;version="[4.0,5)",\
- jakarta.enterprise.util;version="[3.0,4)",\
+ jakarta.enterprise.util;version="[4.0,5)",\
  jakarta.inject;version="[2.0,3)",\
```

Comment:
I don't understand why we would import these packages since we are exporting them.  They may be in one of the bundles and now that we consume that bundle into this bundle and its dependencies are also in this bundle, these imports should not be necessary.

Reply:
The original jakarta.enterprise.cdi-api-4.0.0.jar imports the same packages that it exports and we just copy those same imports and exports

Reply:
Well it exports are from the other bundles besides the cdi-api-4.0.0.  The exports do NOT match cdi-api-4.0.0's exports.  Since we bring in the imported packages into this single jar, these imports should be removed.

Reply:
Both the cdi-api-4.0.0 and lang-model-4.0.0 jars import the same packages that they export ... so we do the same thing here even though the two have been merged together. I am told that it is good practice in order to avoid some potential OSGi wiring issues.

Reply:
You removed too many imports.  Only the ones highlighted should have been removed.

Reply:
Oh yeah the default import would kick in to do the imports.  Forgot about that.  I am going to resolve this one then.